lib/debug/server_dap.rb

```diff
@@ -699,16 +699,15 @@ def process_dap args
         frame = get_frame(fid)
         message = nil
 
-        if frame && (b = frame.binding)
-          b = b.dup
+        if frame && (b = frame.eval_binding)
           special_local_variables current_frame do |name, var|
```

Comment:
@ko1 perhaps this should be `frame` instead?

Reply:
it should be.